src/Grid/Col/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, CSSProperties } from "react";
+import classNames from "../../utils/classNames";
+
+type ColProps = {
+  lg?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | "hide";
+  md?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | "hide";
+  sm?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | "hide";
+  xs?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | "hide";
+  offset?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+  offsetMd?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+  offsetSm?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+  offsetXs?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11;
+  style?: CSSProperties;
+  center?: boolean;
```

Comment:
we don't use center lets remove it

Reply:
but `center` is used in storybook as a prop for `Col` component and that's why I typed it here

Reply:
you should remove it from there instead of adding it here without use

Reply:
or implement the center logic

Reply:
removing it from here and convert `Grid.jsx` to `Grid.tsx` will lead to a crush for the app cause center prop is not typed

Reply:
center logic ? 🤔

Reply:
thats why you should do one pr at a time.